lib/ex_aws/sqs.ex

```diff
@@ -259,10 +262,16 @@ defmodule ExAws.SQS do
   end
 
   defp request(queue, action, params) do
+    action_atom = action
+    |> Macro.underscore
+    |> String.to_atom
```

Comment:
Would it be preferable to have for example `request("", :list_queues, params)` and then

```
defp request(queue, action_atom, params) do
  action = action_atom |> Atom.to_string |> Macro.camelize
end